### PR TITLE
Bump ember-cli-babel@7 in application blueprint

### PR DESCRIPTION
**Pull request by SergeAstapov**

### CODE REVIEW

File: tests/fixtures/addon/npm/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^6.16.0"
+    "ember-cli-babel": "^7.1.2"
```
Comment (Turbo87) [inline tests/fixtures/addon/npm/package.json]: missing comma

File: blueprints/module-unification-app/files/package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "ember-ajax": "^3.1.0",
     "ember-cli": "github:ember-cli/ember-cli",
     "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^6.16.0",
+    "ember-cli-babel": "^7.1.2"
```
Comment (btecu) [inline blueprints/module-unification-app/files/package.json]: Should end with a comma

File: tests/fixtures/module-unification-app/npm/package.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
     "ember-ajax": "^3.1.0",
     "ember-cli": "github:ember-cli/ember-cli",
     "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^6.16.0",
+    "ember-cli-babel": "^7.1.2"
```
Comment (btecu) [inline tests/fixtures/module-unification-app/npm/package.json]: Should end with a comma

File: tests/fixtures/module-unification-app/yarn/package.json [hunk 4]
```diff
@@ -24,7 +24,7 @@
     "ember-ajax": "^3.1.0",
     "ember-cli": "github:ember-cli/ember-cli",
     "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^6.16.0",
+    "ember-cli-babel": "^7.1.2"
```
Comment (btecu) [inline tests/fixtures/module-unification-app/yarn/package.json]: Should end with a comma

File: tests/fixtures/addon/npm/package.json (hunk 1 above)
Comment (SergeAstapov) [reply tests/fixtures/addon/npm/package.json]: @Turbo87 do we need comma here? It's the only entry in `dependencies` section

File: blueprints/module-unification-app/files/package.json (hunk 2 above)
Comment (SergeAstapov) [reply blueprints/module-unification-app/files/package.json]: that's clearly 🤦‍♂️ 
Thanks for catching these up - all occurrences fixed

**Comment by Turbo87** [comment]
@SergeAstapov yeah, AppVeyor seems to be unhappy in general at the moment. Haven't had time yet to figure out why.

**Pull request merged**